_assets/js/netlify/makeHTMLFromBodyContent.js

```diff
@@ -5,7 +5,12 @@ md.options['linkify'] = true;
 const parser = new DOMParser();
 function makeHTMLFromBodyContent(bodyContent, variables, imageData) {
   // Replace any <hr> tags with markdown so they render outside of the nearest <p> tag
-  let content = bodyContent ? bodyContent.replaceAll(/<hr>/g, "***") : '';
+  let content = bodyContent
+    ? bodyContent
+      .replaceAll(/<hr>/g, "***")
+      .replaceAll("{{'", '')
+      .replaceAll("' | relative_url}}", '')
+    : '';
```

Comment:
javascript has a "null coalescing" operator that might be useful here, as it's slightly more extensible than the ternary operator. If you do additional operations to bodyContent that might result in undefined / null, you can add more `?` rather than add more ternary clauses:

```suggestion
  let content = bodyContent?
      .replaceAll(/<hr>/g, "***")
      .replaceAll("{{'", '')
      .replaceAll("' | relative_url}}", '')
    || ''
```